temba/channels/models.py

```diff
@@ -1718,7 +1718,10 @@ def send_vumi_message(cls, channel, msg, text):
         headers = dict(TEMBA_HEADERS)
         headers['content-type'] = 'application/json'
 
-        url = 'https://go.vumi.org/api/v1/go/http_api_nostream/%s/messages.json' % channel.config['conversation_key']
+        api_url_base = channel.config.get('api_url', 'https://go.vumi.org/api/v1/go/http_api_nostream')
```

Comment:
Last thing, since we refer to this URL three different times, let's make it a constant in Channel.